backends/dpdk/backend.cpp

```diff
@@ -53,6 +53,7 @@ void DpdkBackend::convert(const IR::ToplevelBlock *tlb) {
         new P4::TypeChecking(refMap, typeMap),
         // TBD: implement dpdk lowering passes instead of reusing bmv2's lowering pass.
         new BMV2::LowerExpressions(typeMap),
+        new SimplifyTernaryOperator(typeMap, refMap),
```

Comment:
Can we use DoSimplifyExpressions for dpdk?

Reply:
This was initial idea, but it generate unoptimal code, also increases code size , because this pass does many other things apart from dismantling ternary like it consider extern call as side effects, Several optimization is prevented because of this.

Reply:
Ok